core/src/components/button/button.ios.vars.scss

```
@@ -83,7 +83,7 @@ $button-ios-large-height:                             3.1em !default;
 $button-ios-large-border-radius:                      16px !default;
 
 /// @prop - Font size of the large button
-$button-ios-large-font-size:                          20px !default;
+$button-ios-large-font-size:                          1.25rem !default;
```

```suggestion
$button-ios-large-font-size:                          min(1.25rem, 26px) !default;
```

Should I set the max size here to `3px` more than clear?